components/external_server/src/routes/metric.py

```diff
@@ -184,15 +185,17 @@ def post_metric_debt(metric_uuid: MetricId, database: Database):
     return dict(ok=True)
 
 
-@bottle.post("/api/v3/metric/<metric_uuid>/issue/new", permissions_required=[EDIT_REPORT_PERMISSION], pass_user=True)
-def add_metric_issue(metric_uuid: MetricId, database: Database, user: User):
+@bottle.post("/api/v3/metric/<metric_uuid>/issue/new", permissions_required=[EDIT_REPORT_PERMISSION])
+def add_metric_issue(metric_uuid: MetricId, database: Database):
     """Add a new issue to the metric using the configured issue tracker."""
-    reports = latest_reports(database, latest_datamodel(database))
-    report = latest_report_for_uuids(reports, metric_uuid)[0]
+    data_model = latest_datamodel(database)
+    report = latest_report_for_uuids(latest_reports(database, data_model), metric_uuid)[0]
     metric, subject = report.instance_and_parents_for_uuid(metric_uuid=metric_uuid)
+    last_measurement = latest_successful_measurement(database, metric)
+    if not last_measurement:
+        return dict(ok=False, error="Can not create an issue for metric without measurements.")
+    issue_summary, issue_description = create_issue_text(metric, last_measurement, data_model)
```

Comment:
Can't we continue here with a (fake) measurement with an unknown value? That way the user can also create issues for metrics without measurements and we get rid of the if-statement. And get rid of the unit test `test_add_metric_issue_no_measurement` and the feature test `Creating a new issue for metric without measurements`.